opba-protocols/xs2a-protocol-tests/xs2a-bdd-tests-common/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/stages/PaymentRequestCommon.java

```diff
@@ -66,6 +67,42 @@ public SELF fintech_calls_initiate_payment_for_anton_brueckner() {
         return self();
     }
 
+    public SELF fintech_calls_initiate_payment_for_anton_brueckner_tpp_redirect_preferred() {
+        String body = readResource("restrecord/tpp-ui-input/params/anton-brueckner-single-sepa-payment.json");
+        ExtractableResponse<Response> response = withPaymentHeaders(ANTON_BRUECKNER, requestSigningService, PIS, body)
+                .header(TPP_REDIRECT_PREFERRED, "true")
+                .contentType(APPLICATION_JSON_VALUE)
+                .body(body)
+             .when()
+                .post(INITIATE_PAYMENT_ENDPOINT, SEPA_PAYMENT)
+             .then()
+                .statusCode(ACCEPTED.value())
+                .extract();
+
+        updateServiceSessionId(response);
+        updateRedirectCode(response);
+        updateNextPaymentAuthorizationUrl(response);
+        return self();
+    }
+
+    public SELF fintech_calls_initiate_payment_for_anton_brueckner_tpp_redirect_not_preferred() {
+        String body = readResource("restrecord/tpp-ui-input/params/anton-brueckner-single-sepa-payment.json");
+        ExtractableResponse<Response> response = withPaymentHeaders(ANTON_BRUECKNER, requestSigningService, PIS, body)
+                .header(TPP_REDIRECT_PREFERRED, "false")
```

Comment:
Here and elsewhere. Can we define the constant for `false` ?

Reply:
Header addition was removed